### ISSUE TITLE
Laser not firing

### ISSUE THREAD

**Issue by hpolster** [nortd/lasersaur#15]
Our laser is running (no errors on the app and the laser head moves its proper path) but the laser itself does not fire. Any ideas what this could be?

**Comment by allartburns** [comment]
On 12/22/2021 4:52 PM, hpolster wrote:
> Our laser is running (no errors on the app and the laser head moves its 
> proper path) but the laser itself does not fire. Any ideas what this 
> could be?

Lots of things?

Have you power cycled everything?  Turn it off, wait at least a minute 
for the capacitors to drain, turn it back on?

Are you seeing any LED activity on the PSU?  I have clear polycarb 
between the PSU and the bay so I can see the LEDs flicker.

Any LEDs on the PC board flickering?

I had this happen a couple of years ago, it turned out that one of the 
relays on the PCB had died and power never got to the PSU.

--jet



-- 
Jet Townsend
<http://www.flatline.net>
HF: KC3HSO
PGP fingerprint: CBFF BB96 45F0 EC10 9649 030E 12AA 9DBF DC01 847E

**Comment by hpolster** [comment]
Yes, we have power cycled everything. We do not see any leds on the laser generator power supply. Were thinking that might be the problem? 


Hannah

**Comment by allartburns** [comment]
On 12/27/2021 9:37 AM, hpolster wrote:
> Yes, we have power cycled everything. We do not see any leds on the 
> laser generator power supply. Were thinking that might be the problem?


If there's no LED activity on the PSU either it's dead or it's not 
getting power from the PC board.  I would use a VOM to find the power fault.


-- 
Jet Townsend
<http://www.flatline.net>
HF: KC3HSO
PGP fingerprint: CBFF BB96 45F0 EC10 9649 030E 12AA 9DBF DC01 847E
